### PR TITLE
Сreate card for cooperations, and two smaller components for it

### PR DESCRIPTION
**Pull request by MickhaelSh**
image
image

### CODE REVIEW

File: src/components/languages-list-with-icon/LanguagesListWithIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography'
+import LanguageIcon from '@mui/icons-material/Language'
+
+import { styles } from '~/components/languages-list-with-icon/LanguagesListWithIcon.styles'
+import { LanguagesEnum } from '~/types'
+import { FC } from 'react'
```
Comment (Yurenko) [inline src/components/languages-list-with-icon/LanguagesListWithIcon.tsx:7]: Put at the top of the list

File: src/containers/my-cooperations/cooperation-card/CooperationCard.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+import AppCard from '~/components/app-card/AppCard'
+import AppChip from '~/components/app-chip/AppChip'
+import ImgTitleDescription from '~/components/img-title-description/ImgTitleDescription'
+import LanguagesListWithIcon from '~/components/languages-list-with-icon/LanguagesListWithIcon'
+import StatusChip from '~/components/status-chip/StatusChip'
+import TitleWithDescription from '~/components/title-with-description/TitleWithDescription'
+import { getFormatedDate } from '~/utils/helper-functions'
+import { Cooperation } from '~/types'
+import { styles } from '~/containers/my-cooperations/cooperation-card/CooperationCard.styles'
+import { FC } from 'react'
```
Comment (Yurenko) [inline src/containers/my-cooperations/cooperation-card/CooperationCard.tsx:15]: The same here

**Review by Yurenko — APPROVED** [review]

File: src/components/languages-list-with-icon/LanguagesListWithIcon.tsx (hunk 1 above)
Comment (MickhaelSh) [reply src/components/languages-list-with-icon/LanguagesListWithIcon.tsx]: fixed

File: src/containers/my-cooperations/cooperation-card/CooperationCard.tsx (hunk 2 above)
Comment (MickhaelSh) [reply src/containers/my-cooperations/cooperation-card/CooperationCard.tsx]: fixed

**Review by Tolik170 — APPROVED** [review]

**Review by maarekua — APPROVED** [review]

**Pull request merged**